lib/NewDataList/NewDataList.js

```diff
@@ -82,7 +82,7 @@ module.exports = kind({
 		// potentially not in view, and focusing would cause it to
 		// scroll it into view for no reason apparent to the user.
 		// Instead, we just focus the first visible list element.
-		if (event.focusType === 'default' && event.index) {
+		if (event.focusType === 'default' && typeof event.index === 'number') {
```

Comment:
If original issue was aiming to fix the case of focused item is off screen (Not inside of scroller viewort), the can we compare event.index with firstFullyVisibleI and lastFullyVisibleI to filter out that item doesn't need unnecessary scroll in?